src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,160 @@
+  private final PIDController elbowPIDController;
+  private ShuffleboardTab configTab = Shuffleboard.getTab("Arm");
+  //Base Arm Values for Simulation
+  private GenericEntry simBArmAngle = configTab.add("SimBase Arm Angle", 0.0).getEntry();
+  private GenericEntry simBEncoderPos = configTab.add("SimBase Encoder Angle", 0.0).getEntry();
+  private GenericEntry simBOutSet = configTab.add("SimBase Output", 0.0).getEntry();
+  private GenericEntry simBError = configTab.add("SimBase Error", 0.0).getEntry();
+  private GenericEntry simBVoltage = configTab.add("SimBase Motor Voltage", 0.0).getEntry();
+  //Elbow Arm Values for Simulation
+  private GenericEntry simEArmAngle = configTab.add("SimElbow Arm Angle", 0.0).getEntry();
+  private GenericEntry simEEncoderPos = configTab.add("SimElbow Encoder Angle", 0.0).getEntry();
+  private GenericEntry simEOutSet = configTab.add("SimElbow Output", 0.0).getEntry();
+  private GenericEntry simEError = configTab.add("SimElbow Error", 0.0).getEntry();
+  private GenericEntry simEVoltage = configTab.add("SimElbow Motor Voltage", 0.0).getEntry();
+  
+
+  // SIM VALUES:
+  private double baseGearing = 15.;
+  private double elbowGearing = 8.57142857;
+  public static double baseArmLength = Units.inchesToMeters(43.5);
+  public static double elbowArmLength = Units.inchesToMeters(37.5);
+  private double minBAngle = Units.degreesToRadians(45);
+  private double maxBAngle = Units.degreesToRadians(91);
+  private double minEAngle = Units.degreesToRadians(-70);
+  private double maxEAngle = Units.degreesToRadians(89);
+  private double baseArmMass = Units.lbsToKilograms(15); //15
+  private double elbowArmMass = Units.lbsToKilograms(15);
+  private final double baseArmInertia = SingleJointedArmSim.estimateMOI(baseArmLength, baseArmMass);
+  private final double elbowArmInertia = SingleJointedArmSim.estimateMOI(elbowArmLength, elbowArmMass);
+  private double simBCurrentAngle;
+  private double simECurrentAngle;
+
+  private final SingleJointedArmSim baseArmSim = new SingleJointedArmSim(DCMotor.getNEO(2), baseGearing, baseArmInertia, baseArmLength, minBAngle, maxBAngle, baseArmMass, false);
+  private final SingleJointedArmSim elbowArmSim = new SingleJointedArmSim(DCMotor.getNEO(1), elbowGearing, elbowArmInertia, elbowArmLength, minEAngle, maxEAngle, elbowArmMass, false);
+
+  public ArmSubsystem() {
+    motorBaseOne = MotorController.constructMotor(MotorConfig.ArmBaseMotor1);
+    motorBaseTwo = MotorController.constructMotor(MotorConfig.ArmBaseMotor2);
+    motorBaseTwo.follow(motorBaseOne);
+    motorElbow = MotorController.constructMotor(MotorConfig.ArmElbowMotor);
+    //TODO: Place real values for channels
+    motorBaseOneEncoder = new Encoder(0,1);
+    motorElbowEncoder = new Encoder(2,3);
+    motorBaseOneEncoder.setDistancePerPulse(1./baseGearing); //Amount of distance per rotation of the motor, basically a gear ratio measurement
+    motorElbowEncoder.setDistancePerPulse(1./elbowGearing);
```

Comment:
May as well add the conversion to radians in this

Reply:
done, now converted to radians